build.py

```diff
@@ -88,9 +88,9 @@
     'darwin': [ "amd64", "386" ],
     # Windows is not currently supported in InfluxDB 0.9.5 due to use of mmap
     # 'windows': [ "amd64", "386", "arm" ],
-    'linux': [ "amd64", "386" ]
+    'linux': [ "amd64", "386", "arm" ]
 }
-supported_go = [ '1.5.1' ]
+supported_go = [ '1.5.1', '1.4.2' ]
```

Comment:
Yes. This isn't used yet, but I was going to compare version numbers to either prevent builds on unsupported Go versions, or allow for multi-Go-version builds at some point in the future.